dotnet/tools/launcher_net/main.c

```diff
@@ -49,16 +49,18 @@ static void Execute(int argc, char *argv[], const char *manifestDir)
 			printf("argv[%d] = %s\n", i, newargv[i]);
 	}
 #ifdef _MSC_VER
-	exit(_spawnvp(_P_WAIT, newargv[0], newargv));
+	exit((int)_spawnvp(_P_WAIT, newargv[0], newargv));
```

Comment:
Without the cast the warning is:
warning C4244: 'function': conversion from 'intptr_t' to 'int', possible loss of data.